resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
```

Comment:
We need to know the consequence of things.

Say you ping an event which is "content read".

We need to know why that content was read:
- Maybe it is because the user looked at an article (the page -> event hierarchy)
- Maybe it was because the recommended reads component loaded (event -> event hierarchy)
  - and then the user clicked on one of the reads (event -> page hierarchy)
- Maybe it was because a snippet loaded on fastFT and the user clicked read more (event -> event hierarchy)
- Maybe an A/B test changed something on the page and rendered a new component (page -> event -> event)

Without a hierarchy we know none of the above.

Then expand it out to all the different types of event: nav interaction, video interaction, slideshows, interactive graphics. We need to know what happened (counts of events), but also why (hierarchy).